PRS5001-6000/PRS5349Hippolyt.xml

```diff
@@ -44,14 +45,28 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sex="1">
-               <persName>
-                  
-                  Hippolytus
-               </persName>
-               <occupation> wr. </occupation>
+            <person sex="1" sameAs="wd:Q207113">
+               <persName xml:lang="en" xml:id="n1">Hippolytus of Rome</persName>
+               <persName xml:lang="gez" xml:id="n2">ሄጶሊጦስ፡</persName>
+               <persName xml:lang="gez" corresp="#n1">Heṗoliṭos</persName>
+               <persName xml:lang="gez" xml:id="n3">አባ፡ ሊጦስ፡</persName>
+               <persName xml:lang="gez" corresp="#n3">ʾabbā Liṭos</persName>
+               <persName xml:lang="gr" xml:id="n4"> Ἱππόλυτος</persName>
+               <occupation type="ecclesiastic">writer</occupation>
+               <faith type="Greek"/>
+               <birth notAfter="0170"/>
+               <floruit>A Roman presbyter and martyr, one of the most important Christian writers
+               of his time.</floruit>
+               <death when="0235"/>
             </person>
-         </listPerson>
+            <listRelation>
+            <relation name="skos:exactMatch" active="PRS5349Hippolyt" passive="http://www.syriaca.org/person/537"/>
+            <relation name="betmas:formerlyAlsoListedAs" active="PRS5349Hippolyt" passive="PRS5465Hippolyt"/>
+            </listRelation>
+             </listPerson>
+<listBibl type="secondary">
+<bibl><ptr target="bm:Bausi2007Hippolutys"/></bibl>
```

Comment:
there is something wrong with the bm:tag here, please correct to Hippolytus (it is correct in Zotero)